docker-compose.dev.yaml

```diff
@@ -11,7 +11,7 @@ services:
       - content-publishing-service
 
   frequency:
-    image: frequencychain/instant-seal-node:latest
```

Comment:
should we specify the platform?

Reply:
Nice catch; I didn't realize that was an option in a docker-compose file. I'll add it.